manage-data/migrate.md

```diff
@@ -107,86 +115,95 @@ Through the {{es}} reindex API, you can connect your new {{es}} Service deployme
+
+    If your old cluster has multiple repositories configured, identify the repository with the snapshots containing the data that you want to migrate.
 
-3. Add the snapshot repository:
+2. Add the snapshot repository on the new cluster:
 
-    ::::{tab-set}
+    The new cluster must register a snapshot repository that points to the same physical storage location used by the old cluster. This ensures the new cluster can access the existing snapshots.
 
-    :::{tab-item} {{ech}}
+    Considerations:
 
-    From the [console](https://cloud.elastic.co?page=docs&placement=docs-body) of the **new** {{es}} cluster, add the snapshot repository.
+      * If you’re migrating [searchable snapshots](/deploy-manage/tools/snapshot-and-restore/searchable-snapshots.md), the repository name must be identical in the source and destination clusters.
+      * If the old cluster still has write access to the repository, register the repository as read-only to avoid data corruption. This can be done using the `readonly: true` option.
 
-    For details, check our guidelines for:
-    * [Amazon Web Services (AWS) Storage](../deploy-manage/tools/snapshot-and-restore/ec-aws-custom-repository.md)
-    * [Google Cloud Storage (GCS)](../deploy-manage/tools/snapshot-and-restore/ec-gcs-snapshotting.md)
-    * [Azure Blob Storage](../deploy-manage/tools/snapshot-and-restore/ec-azure-snapshotting.md).
+    To configure a custom snapshot repository for your {{ech}} or {{ece}} deployment, follow the steps for the storage provider used by your existing repository:
 
-    If you’re migrating [searchable snapshots](../deploy-manage/tools/snapshot-and-restore/searchable-snapshots.md), the repository name must be identical in the source and     destination clusters.
+    * **Amazon Web Services (AWS) Storage**
+        * [Store credentials in the keystore](/deploy-manage/tools/snapshot-and-restore/ec-aws-custom-repository.md#ec-snapshot-secrets-keystore)
+        * [Create the repository](/deploy-manage/tools/snapshot-and-restore/ec-aws-custom-repository.md#ec-create-aws-repository)
+    * **Google Cloud Storage (GCS)**
+        * [Store credentials in the keystore](/deploy-manage/tools/snapshot-and-restore/ec-gcs-snapshotting.md#ec-configure-gcs-keystore)
+        * [Create the repository](/deploy-manage/tools/snapshot-and-restore/ec-gcs-snapshotting.md#ec-create-gcs-repository)
+    * **Azure Blob Storage**
+        * [Store credentials in the keystore](/deploy-manage/tools/snapshot-and-restore/ec-azure-snapshotting.md#ec-configure-azure-keystore).
+        * [Create the repository](/deploy-manage/tools/snapshot-and-restore/ec-azure-snapshotting.md#ec-create-azure-repository).
+    
+    ::::{important}
+    Although the previous instructions are focused on {{ech}}, you should follow the same steps for {{ece}} by configuring the repository directly **at the deployment level**.
```

Comment:
hmm, sounds like we should make it clear that this is possible in the snapshot docs. do you think it makes sense to open an issue to add new applies tags and notes with a use case?

Reply:
do you mean configuring repositories at deployment level in ECE? Totally agreed, we should improve that visibility.
It's not a very common use case but a perfectly valid one, and I'm sure some customers are already doing that, as if for example they want a repo dedicated to searchable snapshots they have to create it at deployment level, and not managed by ECE.

We should create an issue to add this info and link from ECE docs to ECH docs + making sure the ECH docs in this are also applies_to ECE + explain it a bit.

Reply:
Created https://github.com/elastic/docs-content/issues/1494 as a follow up issue for this.